test/compile/scale/type.test.ts

```diff
@@ -140,11 +140,17 @@ describe('compile/scale', () => {
         })
       );
 
-      it('should return time for all time units.', () => {
-        for (const timeUnit of TIMEUNITS) {
+      it('should return time for all non-utc time units.', () => {
+        for (const timeUnit of TIMEUNITS.filter(t => !isUTCTimeUnit(t))) {
           expect(scaleType({}, Y, {type: 'temporal', timeUnit: timeUnit}, 'point')).toEqual(ScaleType.TIME);
         }
       });
+
+      it('should return utc for all utc time units.', () => {
```

Comment:
It shouldn’t?

Reply:
When would it not?

Reply:
The issue again may be double shifting, I think.